src/emulator/gui/src/gui.cpp

```diff
@@ -167,18 +167,24 @@ static void init_icons(HostState &host) {
         int32_t width = 0;
         int32_t height = 0;
         vfs::FileBuffer buffer;
+        const std::string default_icon = "../data/icon/Vita3K.png";
 
         vfs::read_app_file(buffer, host.pref_path, game.title_id, "sce_sys/icon0.png");
         if (buffer.empty()) {
-            LOG_WARN("Icon not found for title {}, {}.", game.title_id, game.title);
-            continue;
+            if (fs::exists("data/icon/Vita3K.png")) {
+                LOG_INFO("Icon default found for title {}, {}.", game.title_id, game.title);
+                vfs::read_file(VitaIoDevice::UX0, buffer, host.pref_path, default_icon);
+            } else {
+                LOG_WARN("Icon default not found for title {}, {}.", game.title_id, game.title);
```

Comment:
adjective goes first, noun goes after

```suggestion
                LOG_WARN("Default icon not found for title {}, {}.", game.title_id, game.title);
```